src/neo/Wallets/Wallet.cs

```diff
@@ -250,12 +251,12 @@ public Transaction MakeTransaction(TransferOutput[] outputs, UInt160 from = null
             else
             {
                 if (!Contains(from))
-                    throw new ArgumentException($"The address {from.ToString()} was not found in the wallet");
+                    throw new ArgumentException($"The address {from} was not found in the wallet");
```

Comment:
So as https://github.com/neo-project/neo-node/pull/628#issuecomment-667964407, we don't need this judgment.

Reply:
Done